### PR TITLE
feat: implement select base token page

### PR DESCRIPTION
**Pull request by Tonyverse-ops**
## Description
feat: implement select base token page

## Related Issue(s)
#3 

- [x] Read the [contributing docs](../CONTRIBUTING.md) (if this is your first contribution)
- [x] Verified this is not a duplicate of any existing pull request

- [x] Performed a self-review of the code
- [x] Linked the issues which this PR resolves
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Documented the changes in code (API docs will be generated automatically)

### CODE REVIEW

**Comment by Oshioke-Salaki** [comment]
sync your fork and update your pr. The layout background on the layout has been changed.

**Comment by Tonyverse-ops** [comment]
@Oshioke-Salaki kindly review and merge :)

**Comment by Oshioke-Salaki** [comment]
please create a select-base-token route so i can be able to preview your component.

**Comment by Oshioke-Salaki** [comment]
Please make it responsive

**Comment by Tonyverse-ops** [comment]
@Oshioke-Salaki implemented required change

**Comment by Oshioke-Salaki** [comment]
Build is failing

**Comment by Tonyverse-ops** [comment]
hey @Oshioke-Salaki kindly review :)

File: app/components/select-base-token.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import usdt from "../../public/usdt.svg";
+import usdc from "../../public/usdc.svg";
+import checked from "../../public/checked.svg";
+import unchecked from "../../public/unchecked.svg";
+
+export function SelectBaseToken() {
+  const [active, setActive] = useState<"USDT" | "USDC">("USDT");
+
+  return (
+    <div className="flex justify-center items-center mt-10 w-full px-4 bg-gradient-to-b from-[#08001F] to-[#040010]">
```
Comment (Oshioke-Salaki) [inline app/components/select-base-token.tsx:15]: No need for this gradient.

File: app/components/select-base-token.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      <div className="flex relative justify-center items-center rounded-3xl border border-[#170F2E] text-center w-full max-w-[50rem] flex-col bg-[#08001F]/90 backdrop-blur-sm p-6 mx-4">
```
Comment (Oshioke-Salaki) [inline app/components/select-base-token.tsx:16]: `bg-[#08001F]/90 backdrop-blur-sm' no need for this

**Review by Oshioke-Salaki — CHANGES_REQUESTED** [review]

**Comment by Oshioke-Salaki** [comment]
Please sync your fork and update your pr.

**Comment by Oshioke-Salaki** [comment]
Also noticed that your page doesn't take up the full page of the screen. Please fix that.
Screenshot 2024-11-27 at 15 15 20

**Comment by Tonyverse-ops** [comment]
hey @Oshioke-Salaki kindly review :)

**Comment by Oshioke-Salaki** [comment]
The background isn't visible on mobile screens and tablets.

**Comment by Oshioke-Salaki** [comment]
Screenshot 2024-11-27 at 16 56 22
Please add some space between the component and the header on mobile screens. Also the close button and the text are overlapping.

**Comment by Tonyverse-ops** [comment]
@Oshioke-Salaki kindly review 
@Oshioke-Salaki

**Comment by Oshioke-Salaki** [comment]
lgtm 🚀

**Pull request merged**